### ISSUE TITLE
Special shows with special music

### ISSUE THREAD

**Issue by theshillito** [very-scary-scenario/nkd.su#46]
Every now and again I will have a special show featuring music not normally in the NekoDesu music library. The typical example so far would be B-sides, but remix and image song specials may also happen.

As the site is currently, I would have to import these tracks into the main library, and then hide them afterwards. I propose having multiple spin-off libraries where I can upload an alternative playlist from iToons and it populates its own section on the site. The tracks will only become visible when a special show is planned (perhaps adapting #39 to add a show type). The tracks should probably be a different colour to differentiate them from standard tracks in search (could also implement filters to only search the special tracks).

Once a show ends, the tracks will be in the playlist archives, but cannot be voted for again (so clicking on them will need to be disabled).

So in short:
1. Upload a playlist of special tracks into its own table, tagged with type (remix, image, etc)
2. Tracks are hidden until special show is the following week.
3. When shown, special track buttons are a different colour
4. Special tracks can be searched either alongside standard tracks or separate.
5. At the end of the show, tracks are hidden again except in archives, but are not clickable in them.

I think that's everything. Let me know if there's anything else you need.
